cdap-ui/app/cdap/components/RulesEngineHome/RuleBookDetails/RuleBookDetails.scss

```diff
@@ -98,6 +97,13 @@ $rule-book-create-fields-required-color: $cdap-red-text;
       margin-top: -10px;
     }
   }
+
+  .rule-book-content {
+    height: calc(100% - 100px);
```

Comment:
nit: I guess this `100px` is the height of `.rule-book-name-header`? Let's make it a variable so we only have to modify it once if we change it later